test/index.test.ts

```diff
@@ -20,3 +20,4 @@ import './server/column-privileges'
 import './server/materialized-views'
 import './server/table-privileges'
 import './server/typegen'
+// import './server/ssl'
```

Comment:
typo ?

Reply:
Intentionally omitted, since setting up the ssl key/certs was a pain. But I'll attempt it one last time